_layouts/default.html

```diff
@@ -33,7 +33,7 @@
               {% assign location = site.data.locations[event.location][page.lang] %}
               <address class="upcoming-info__location">
                 <label class="upcoming-info__show-map" for="{{event.sortDate}}map" title="{{commonStrings.showMap}}">
-                  {{location.name}}, {{location.address}}{% if event.time %}, {{event.time}}{% endif %}
+                  {{location.name}}, {{location.address}}
```

Comment:
right, but there's one more  in '_event' include: 
https://github.com/listochkin/kyivjs/blob/gh-pages/_includes/event.html#L5
should I change the PR and leave both?